spec/rails_helper.rb

```diff
@@ -95,6 +110,35 @@ def get(path, headers: nil, sign_with: nil, **args)
     stub_jsonld_contexts!
   end
 
+  config.before :suite do
+    if RUN_SYSTEM_SPECS
+      Webpacker.compile
+      streaming_server_manager.start(port: STREAMING_PORT)
+    end
+  end
+
+  config.after :suite do
```

Comment:
The `before` had a conditional, but this does not?

Reply:
If the streaming server is not started then this is a no-op, so I chose to not add a conditional here